src/main/java/com/example/petbuddybackend/entity/animal/AnimalTakenCareOf.java

```diff
@@ -13,15 +13,15 @@
 @Builder
 @Entity
 @Table(
-        uniqueConstraints = { @UniqueConstraint(columnNames = { "caretaker_email", "animalType" }) }
+        uniqueConstraints = { @UniqueConstraint(columnNames = { "caretakerId", "animalType" }) }
 )
 public class AnimalTakenCareOf {
```

Comment:
```suggestion
public class CaretakerOffer {
```

Jednak bardziej by mi pasowało nazwanie tego ofertą opiekuna, wtedy można dodać zwierze jakim może się opiekować i od razu do zwierzęcia (i w przyszłości jakichś bardziej szczegółowych informacji o zwierzęciu) informacje na temat stawki.

Reply:
done